### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/24/day23.html

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#32] [Comments]
# https://mnfienen.github.io/talulat/2023/08/24/day23.html

Day 23 - Downhill to the most infamously famous tidal bore of the BAY OF FUNDY!

https://mnfienen.github.io/talulat/2023/08/24/day23.html

**Comment by christianlangevin** [comment]
I've been hearing about the Bay of Fundy for many years.  Cool to see you made it.  Fantastic journey.

**Comment by mnfienen** [comment]
Thanks @langevin-usgs! It was great to make it! I think we misjudged exactly where to be to see the actual bore (and the timing) but cool nonetheless
